lib/php-extension/Aikido.cpp

```diff
@@ -91,8 +93,10 @@ PHP_MINFO_FUNCTION(aikido) {
 
 static const zend_function_entry ext_functions[] = {
     ZEND_NS_FE("aikido", set_user, arginfo_aikido_set_user)
-        ZEND_NS_FE("aikido", should_block_request, arginfo_aikido_should_block_request)
-            ZEND_FE_END};
+    ZEND_NS_FE("aikido", should_block_request, arginfo_aikido_should_block_request)
+    ZEND_NS_FE("aikido", auto_block_request, arginfo_aikido_auto_block_request)
```

Comment:
Should we use a name like `\aikido\__do_not_use_auto_block_request`? e.g. if people find the function using `get_defined_functions`